iroha/src/torii/utils.rs

```diff
@@ -0,0 +1,96 @@
+use std::convert::TryInto;
+
+use iroha_version::scale::DecodeVersioned;
+use parity_scale_codec::Encode;
+use warp::{hyper::body::Bytes, reject::Reject, reply::Response, Filter, Rejection, Reply};
+
+use super::VerifiedQueryRequest;
+
+/// Structure for empty response body
+pub struct Empty;
+
+impl Reply for Empty {
+    fn into_response(self) -> Response {
+        Response::new(vec![].into())
+    }
+}
+
+/// Structure for response in scale codec in body
+pub struct Scale<T>(pub T);
+
+impl<T: Encode + Send> Reply for Scale<T> {
+    fn into_response(self) -> Response {
+        Response::new(self.0.encode().into())
+    }
+}
+
+/// Adds state to filter
+pub fn add_state<State: Send + Clone>(
+    state: State,
+) -> impl Filter<Extract = (State,), Error = Rejection> + Clone + Send {
+    warp::any().and_then(move || {
+        let state = state.clone();
+        async move { Ok::<_, Rejection>(state) }
+    })
+}
+
+pub mod body {
+    use super::*;
+
+    //pub fn scale<T: Decode>() -> impl Filter<Extract = (T,), Error = Rejection> + Copy {
```

Comment:
We should not have any commented code. If it is used or planned to be used, then please uncomment, otherwise we can simply remove it.

Reply:
I wasnt sure whether we will or wont use combinator like that in future. If we have policy that we shouldnt have commented code, then it can be removed i guess

Reply:
Yep, better to remove code when we are not sure